descqa/shear_test.py

```diff
@@ -32,6 +32,8 @@ def __init__(self,
                  dec='dec',
                  e1='shear_1',
                  e2='shear_2_phosim',
+                 mag='restframe_extincted_sdss_abs_magr',
```

Comment:
is `mag` supposed to be the column name of the magnitude? If so, the default value should set to something in the schema, rather than a catalog-specific native quantitiy name

Reply:
this is now set to Mag_true_r_sdss_z0